smartapps/konnected-io/konnected-alarm-panel-connect.src/konnected-alarm-panel-connect.groovy

```diff
@@ -105,7 +105,7 @@ def pageConfiguration() {
   dynamicPage(name: "pageConfiguration") {
     configuredAlarmPanels.each { alarmPanel ->
       section(hideable: true, "AlarmPanel_${alarmPanel.mac[-6..-1]}") {
-        for ( i in [1, 2, 5, 6, 7, 8]) {
+        for ( i in [1, 2, 5, 6, 7]) {
```

Comment:
I read on the forum that some ppl had success with pin 8 as an alarm output, but not as an input. I want to experiment with different pins a little bit more, so we may end up having to revisit this. For example I think we can use the TX and RX pins for sensors too but I need to verify this.